client/src/Home/Home.js

```diff
@@ -34,6 +34,9 @@ function Home() {
               ))}
             </select>
           )}
+        <p>
```

Comment:
This p tag really should have been indented one more to line up with the code above

Reply:
Thanks for the comment! I've made the change!